recipes/pymc/meta.yaml

```diff
@@ -13,15 +13,17 @@ source:
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: python setup.py install
 
 requirements:
   build:
-    - python --single-version-externally-managed --record record.txt
-    - setuptools
-    - numpy>=1.8
+    - python
+    - numpy >=1.8
+    - gcc # [not win]
+    skip: True # [win]
```

Comment:
This skip belongs to the `build` section and not `requirements/build`.

BTW, see https://github.com/conda-forge/staged-recipes/pull/2233